### PR TITLE
Add "plane_crash" map by minetevc

### PR DESCRIPTION
**Pull request by JostP**
(This pull request was generated by a script)

### CODE REVIEW

**Comment by JostP** [comment]
Needs screenshot

**Comment by JostP** [comment]
Map doesn't work. Bases' coordinates are wrong

**Comment by olliy1or** [comment]
also needs a license

**Comment by Sirius-Black4** [comment]
I have licensed the latest map which is in the forums u can update it from there, plus u can see my latest post on the forums and that i have sent screenshots too.

**Comment by Avyukt1** [comment]
You need to select Z=0 while placing the barriers.
If you do X=0, you will get wrong initials of the bases and the barrier will be crashed.

**Comment by Sirius-Black4** [comment]
Avyukt 
i'm not sure if that's true because i think that its not the purpose of the x = 0 and z = 0,these are just the way the barrier is placed,if u put x = 0 the barrier will face a different direction and same if you put z = 0.

**Comment by Sirius-Black4** [comment]
Is there still some problems with the map or is it all fixed(?), if yes tell me how to fix it.

**Comment by Avyukt1** [comment]
@Sirius-Black4 I had got a lots of errors while making my map.

**Comment by Sirius-Black4** [comment]
Is there anything i can do to fix the coords?

**Comment by Avyukt1** [comment]
Lets tell the map maker to redo the map with the barriers z=0. Lets see if it works?

**Comment by Avyukt1** [comment]
Also a screenshot is required to use the map.

File: plane_crash/map.conf [hunk 1]
```diff
@@ -0,0 +1,13 @@
+team.2.color = blue
+team.2.pos = (434,51,-1)
+team.1.pos = (478,52,163)
+team.1.color = red
+team.1 = red
+author = minetevc
+rotation = z
+h = 140
+team.2 = blue
+name = Plan Crash In Early December.Search for a tresure in your base.
```
Comment (Panquesito7) [inline plane_crash/map.conf:10]:
```suggestion
name = Plan Crash In Early December. Search for a treasure in your base.
```

Comment (JostP) [reply plane_crash/map.conf:10]: `Search for a tresure in your base.` looks like a hint

**Review by Avyukt1 — CHANGES_REQUESTED** [review]
Instead of using the line:
`name = Plan Crash In Early December. Search for a treasure in your base.`
use
`name = Plane Crash In Early December.`
`hint = Search for a treasure in your base.`

**Comment by Sirius-Black4** [comment]
So is there a way i can edit the file using Github or do i have to edit the .conf file , correct it and re-upload it?

**Comment by Sirius-Black4** [comment]
Btw. who all are the people that have write access to merge pull requests?

**Comment by Avyukt1** [comment]
@JostP has the access to edit these file.

**Comment by Sirius-Black4** [comment]
I gess i will have too redo the whole map.

**Comment by Avyukt1** [comment]
Yes, If have not emerged the are.

**Comment by Sirius-Black4** [comment]
yes, i haven't emerged it properly. Btw how do i remove barriers because other wise il need to redo every thing.

**Comment by Avyukt1** [comment]
The steps of redoing the map are:
- Download the ctf_map modpack and the WorldEdit modpack.
- Create a new world with the these mods enabled.
- In the chat, do /gui and then click 'Player Pos'. Then click 'From We' and 'To We'.
- After that, choose z=0 and place the barriers.
- Then place the two flags.
- do /gui and in 'initial stuff' give at least a pickaxe. Or type 'default:pick_stone''
- Then type your username from the minetest ctf server as the author of the map.
- Then simply click export.
-  Take a screenshot of the map and host the files in this Pull Request.

**Comment by Avyukt1** [comment]
Remember.
Don't place the flags before placing the barrier.
The **_very fist_** thing is to emerge the area.
Go to https://minetest.rubenwardy.com/ctf for help.

**Comment by Avyukt1** [comment]
Follow the instructions and don't change the sequence of the list in the above message.
Do it step by step.

**Comment by Sirius-Black4** [comment]
You said that The very first thing is to emerge the area. Do you mean even before placing the barriers and just after making the world?

**Comment by Avyukt1** [comment]
Yes. True

**Comment by Avyukt1** [comment]
No
I am the very first thing.
Even before making the world

**Comment by Sirius-Black4** [comment]
Ok, i've redone the map , this is a whole new one and is very similar except it wont be in the chilly altitudes but more in a greenish mountain area.This time i've made it more fun by adding ores like iron , coal ,diamond and mese in concentrated areas in both sides of the team(equally).The mese and diamond will be found in your own base hidden,the iron will be in some ruins you will have to find.To make it even more fun i've also made a broken airplane crashed in the middle of the map.....and just wanted to know if i will have to make a new PR to get this one published?
Thanks for the support, ive learnt a lot :)  .

**Comment by Avyukt1** [comment]
@JostP 
Please take his files and add it to this Pull Request.

**Comment by Avyukt1** [comment]
> Ok, i've redone the map , this is a whole new one and is very similar except it wont be in the chilly altitudes but more in a greenish mountain area.This time i've made it more fun by adding ores like iron , coal ,diamond and mese in concentrated areas in both sides of the team(equally).The mese and diamond will be found in your own base hidden,the iron will be in some ruins you will have to find.To make it even more fun i've also made a broken airplane crashed in the middle of the map.....and just wanted to know if i will have to make a new PR to get this one published?
> Thanks for the support, i've learnt a lot :) .

@Sirius-Black4 
Make a new GitHub repository with these a the
`map.conf`
`map.mts`
and we need a screenshot of the map.
press F12 while flying to capture a screenshot.
add the screenshot.png file in your repository.
@JostP will make the changes.

**Comment by Sirius-Black4** [comment]
Ok ,can someone make a new PR for me idk how to make one, i have sent the screenshots and .zip in the forums.

**Comment by Avyukt1** [comment]
Share me the link please

**Comment by Sirius-Black4** [comment]
Link for what?
This is the link for the screenshots of the new map - https://postimg.cc/gallery/DhcdnY7

**Comment by Avyukt1** [comment]
Where are your files?

**Comment by Avyukt1** [comment]
We need the files for the new Pull Request

**Comment by Avyukt1** [comment]
At least tell me which topic for the minetest forums have you given the map .zip file?

**Comment by Sirius-Black4** [comment]
This is a whole new one and is very similar except it wont be in the chilly altitudes but more in a greenish mountain area.This time i've made it more fun by adding ores like iron , coal ,diamond and mese in concentrated areas in both sides of the team(equally).The mese and diamond will be found in your own base hidden,the iron will be in some ruins you will have to find.To make it even more fun i've also made a broken airplane crashed in the middle of the map.....and just wanted to know if i will have to make a new PR to get this one published?Screenshots can be viewed here - https://postimg.cc/gallery/DhcdnY7
Thanks for the support, ive learnt a lot :) .
© 2020. This work is licensed under a CC BY 4.0 license.
[PlaneCrashCTF.zip](https://github.com/MT-CTF/maps/files/5535462/PlaneCrashCTF.zip)

**Comment by Avyukt1** [comment]
@Sirius-Black4 
-You have placed the flags before placing the barrier.
-You have made a big mess.
-You also emerged the area after building.

**Comment by Sirius-Black4** [comment]
No, i haven't placed the flag before placing the barriers , and what do you mean i have made a big mess?

**Comment by Avyukt1** [comment]
The barriers are invisible while the barriers you placed are on the far off.
We also cannot get the items.

**Comment by Avyukt1** [comment]
It should be
`initial_stuff = default:pick_steel,default:torch 10`
instead of what you wrote.

**Comment by Sirius-Black4** [comment]
well i dont understand what do you mean the barriers are invisible, can you please send a screenshot.....

**Comment by Avyukt1** [comment]
> well i dont understand what do you mean the barriers are invisible, can you please send a screenshot.....
I can give you a video for that

**Comment by Sirius-Black4** [comment]
Yes can you send something so i can see how it looks for you.

**Comment by Avyukt1** [comment]
You can try testing it.
go to https://github.com/MT-CTF/capturetheflag/ to download the subgame.

**Comment by Avyukt1** [comment]
open http://www.filedropper.com/minetest530opengl4302020-11-1313-48-29
and download the video

**Comment by Sirius-Black4** [comment]
Thanks i get it, idk whats causing it to get you back to the base because i did every thing you told me to do except for merging it after placing barriers which i suppose **can not** be the reason for you going back to your base.

**Comment by Avyukt1** [comment]
> Thanks i get it, idk whats causing it to get you back to the base because i did every thing you told me to do except for merging it after placing barriers which i suppose **can not** be the reason for you going back to your base.

Yes, It happens.
But If you are in the other team. you can dig underground and cross the barrier.

**Comment by Avyukt1** [comment]
And Yes.
You have to merge the area even before building the place and placing barriers.

**Comment by Avyukt1** [comment]
Anyway,
I liked the map.
Good map for the server' Let @JostP test it.
Till then, you debug the map.
One more thing
I am `smugler5`

**Comment by Avyukt1** [comment]
I'll give you the video how to make the map.
It will tell you the proper sequence

**Comment by Sirius-Black4** [comment]
Thanks alot man, your a great help.

**Comment by Avyukt1** [comment]
You can see the tutorial at https://1drv.ms/v/s!AlEgFl9GC_oKiQUiX-Bhr83cmQPS?e=RaHspN

**Comment by Sirius-Black4** [comment]
Thanks alot, this is exactly what i did except i emerged it wrong.

**Comment by Avyukt1** [comment]
Redo the map a and give the files.
I'll create a new Pull Request

File: plane_crash/map.conf [hunk 2]
```diff
@@ -7,7 +7,7 @@ author = minetevc
 rotation = z
 h = 140
 team.2 = blue
-name = Plan Crash In Early December.Search for a tresure in your base.
+name = Plan Crash In Early December. Search for a treasure in your base.
```
Comment (xenonca) [inline plane_crash/map.conf:10]: I think it was supposed to be `Plane Crash` right?
Also `Search for a treasure in your base` should be a `hint` I guess

**Review by Avyukt1 — CHANGES_REQUESTED** [review]

**Comment by Avyukt1** [comment]
@JostP  @xenonca 
Well, minetevc/[@Sirius-Black4 ](https://github.com/Sirius-Black4) is not online from a last few months. There are also bugs in this map. He hasn't fixed the bugs nor he has even been online in these day. We need to make the decision faster because other maps are still left for reviewing. I don't think we will get a reply for him. If we get, that's actually good, but if we don't we will have to temporarily close this PR.

**Comment by Sirius-Black4** [comment]
Hi well i am online mostly but i cant play often these days, i think i will be able to fix the bugs by this Friday or so. I will need to redo the whole map probable.

**Comment by JostP** [comment]
I think there is an updated version on MT forum, I can update PR today

**Comment by Avyukt1** [comment]
@Sirius-Black4  Thanks a lot for coming and replying back ;)

**Comment by JostP** [comment]
Updated.
@Avyukt1 you can test it if you want.

**Comment by JostP** [comment]
Map is exported wrongly it is cutted at the borders
Zaslonska slika 2020-12-16 14-11-57

**Comment by Avyukt1** [comment]
The barrier doesn't disappear and its not in the middle of the map.

File: plane_crash/map.conf [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+team.2.pos = (54,-5,-11)
+team.1.pos = (3,1,96)
+team.1.color = red
+team.1 = red
+author = minetevc
+rotation = z
+h = 140
+team.2 = blue
+name = Plane Crash In Mid-Summer
+initial_stuff = default:pick:steel,default:torches:10
```
Comment (Avyukt1) [inline plane_crash/map.conf:11]: There is a bug in initial stuff.
```suggestion
initial_stuff = default:pick_steel,default_torch 10
```

**Review by Avyukt1 — CHANGES_REQUESTED** [review]

**Comment by Avyukt1** [comment]
bug
Man, that barrier isn't working properly, I can noclip through it.

**Comment by Avyukt1** [comment]
@JostP I think the `WIP` label is needed.

**Comment by JostP** [comment]
> @JostP I think the `WIP` label is needed.
> And, we should convert this into a draft.

yes, I will add WIP, but I don't think draft is needed.

**Comment by JostP** [comment]
Will re-open as soon as map is updated.

**Review by Sirius-Black4 — COMMENTED** [review]
> Will re-open as soon as map is updated.

Ok sure.

**Pull request closed without merging**